packages/scripts/import/convertJsonRowToEntryFormat.ts

```diff
@@ -61,6 +61,10 @@ export function convertJsonRowToEntryFormat(
     delete entry.xs;
   }
 
+  if (entry.ps) {
+    entry.ps = entry.ps.map((pos) => pos.trim());
+  }
+
```

Comment:
@jacob-8 is this the optimal solution?

Reply:
Yes, that is the necessary code. Now I've cleaned it up into this: 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/7559478/206812968-5b307933-16bc-4cf0-8143-562197235668.png">

Reply:
I want to start relying more on little tests like this rather than just checking the output of the big test, as there are too many things to check in the big test to know if something breaks.

Reply:
I cleaned up the big test a little. There's a lot more that could be done in the big convert `convertJsonRowToEntryFormat` to clean it up, but at least we are making small steps in the right direction:

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/7559478/206813864-e616410c-d205-4f65-a5d9-443d1f1978d6.png">

It's never possible to do a complete clean sweep, but at least we can improve things a little each PR.